components/financial_assistance/app/models/financial_assistance/applicant.rb

```diff
@@ -1338,7 +1338,12 @@ def set_evidence_outstanding(evidence, desired_due_date = nil)
     end
 
     def set_evidence_to_negative_response(evidence)
-      evidence.negative_response_received! if evidence.may_negative_response_received?
+      return unless evidence.may_negative_response_received?
+
```

Comment:
set evidence to nil when transitioned to nrr.
added specs fro the change, lgtm.